modules/scheduler/pre-existing-gke-cluster/main.tf

```diff
@@ -19,3 +19,57 @@ data "google_container_cluster" "existing_gke_cluster" {
   project  = var.project_id
   location = var.region
 }
+
+data "google_client_config" "default" {}
+
+provider "kubernetes" {
```

Comment:
Even if you can get away with using kubernetes provider here, I would use the same kubectl as gke-cluster module for similarity.

Reply:
done